lib/apisauce.ts

```diff
@@ -223,11 +225,11 @@ export const create = config => {
     }
 
     // after the call, convert the axios response, then execute our monitors
-    const chain = pipe(
-      convertResponse(toNumber(new Date())),
-      // partial(convertResponse, [toNumber(new Date())]),
-      runMonitors,
-    )
+    const chain = async (axiosResult) => {
```

Comment:
@navneetccna So the issue here is that moving the `toNumber(new Date())` inside this function definition has delayed it's execution resulting in it not being called until after the request has been made giving us 0 duration. The easiest way to fix this is to go back to using `pipe` but we need promises so `pipeP`.

```suggestion:+5
const chain = pipeP(
  convertResponse(toNumber(new Date())),
  runMonitors,
)
```